learn/advanced/known_limitations.md

```diff
@@ -100,3 +100,9 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
 **Limitation:** By default, Meilisearch returns up to 1000 documents per search.
 
 **Explanation:** Meilisearch limits the maximum amount of returned search results to protect your database from malicious scraping. You may change this by using the `maxTotalHits` property of the [pagination index settings](/reference/api/settings.md#pagination-object). `maxTotalHits` only applies to the [search route](/reference/api/search.md) and has no effect on the [get documents endpoint](/reference/api/documents.md#get-documents).
+
+## Large datasets and internal errors
+
+**Limitation:** Meilisearch might throw an internal error when indexing large datasets.
+
+**Explanation:** Indexing datasets over 2GB in size can result in Meilisearch opening too many file descriptors. Depending on your system, this might reach your system's resource usage limits and trigger an internal error. Use `ulimit` or a similar tool to increase resource consumption limits.
```

Comment:
- The issue mentions 2GB for CSV and 3.5GB for JSON. Should we mention this as well? 
- Should we add a link or maybe a line or two on what `ulimit` is and how to use it?

Reply:
> The issue mentions 2GB for CSV and 3.5GB for JSON. Should we mention this as well?

I think mentioning the two is not super useful (they are the same dataset, but in different formats), but you're right that we should make it a bit more explicit.